app/src/main/java/de/westnordost/streetcomplete/data/visiblequests/TeamModeQuestFilter.kt

```diff
@@ -59,4 +59,8 @@ import javax.inject.Singleton
     fun removeListener(listener: TeamModeChangeListener) {
         listeners.remove(listener)
     }
+
+    companion object {
+        const val MAX_TEAM_SIZE = 12
```

Comment:
I thought you didn't want to hardcode this but return the colors array size instead?

Reply:
Well, now that the Colors object is gone (converted into view), it seemed wrong to me to have this information sourced from a view class. So I thought I put it where the TeamMode logic has its core.

Reply:
Fixed in #2613.